### ISSUE TITLE
[Bug]: SB3 Functional draws + chipped

### ISSUE THREAD

**Issue by wendictive** [FTBTeam/FTB-Modpack-Issues#1650]
### Modpack

FTB Stoneblock 3

### Modpack version

1.0.2

### Log Files

https://justpaste.it/1nyg1

### Describe the bug

I was making custom draws with black concrete and white wool chipped dont think any Chipped blocks can be used in functional draw frames

### Steps to reproduce

in crafting table (stone one) plase black concrete - a white chipped wool - and a framed draw

### Expected behaviour

to get an awsome looking draw

**Comment by Sunekaer** [comment]
https://github.com/Buuz135/FunctionalStorage/issues/111

**Comment by ukmcplyr** [comment]
Adding in case it helps anyone else.

I ran into this today on a server where attempting to craft a framed compacting drawer with two Chipped blocks crashed as above, but because I did the craft in AE2, the recipe doesn't get cleared in eg Wireless Crafting Terminal and you just get stuck in a crash loop every time you reconnect and access AE2 via the same terminal again.

I've restored from a backup but is there a better option? A punt on enabling `clearGridOnClose` in `config/appliedenergistics2/client.json` client-side and trying to close the terminal very quickly didn't hep.
